### DISCUSSION TITLE
How to set default battery management options for Mac OS

### DISCUSSION THREAD

**Discussion by hyn2028** [AppHouseKitchen/AlDente-Battery_Care_and_Monitoring#293] [Q&A] [answered]
Hi!

I've tried using the application, and it works perfectly the way I want it to. Thank you. I was wondering how the built-in battery life management option of Mac OS is recommended. I read that in the README file, but I don't understand exactly which option is. As far as I know, the Mac OS has two built-in battery life management options: "Optimized battery charging" and "Manage battery longevity". What are the recommendations for each option?

Thank you.

**Comment by MatthiasKerbl** [comment, marked as answer]
Hi @hyn2028,

You should definitely deactivate "Optimized battery charging" and you can disable "Manage battery longevity" as well but you don't have to.
